src/components/LocationSelector/EventCreation.jsx

```diff
@@ -100,17 +105,19 @@ const EventCreation = ({ label, t }) => {
     }, [downloadUrl]);
 
     const handleSubmit = async (e) => {
+        e.preventDefault();
         if (!eventData.address || !eventData.title) {
             alert("Please fill out all the required fields.");
             return;
         }
+
         const eventDataWithLocation = createEventDataWithLocation();
         const result = await addDocument(eventDataWithLocation, eventData);
+
         if (result.success) {
             console.log("Event added successfully with ID:", result.id);
-            window.alert("Event added successfully with ID:", result?.title);
-
-            router.push("/events");
+            window.alert(`Event added successfully with ID: ${result.id}`);
```

Comment:
I think its better to either have a visually better indicator that the event have been added like using [Toast](https://react.flatifycss.com/?path=/docs/components-toast--default) or either removing them.

Reply:
I am going to use it. I was trying to test it